docs/en_US/TrialExample/Pix2pixExample.rst

```diff
@@ -0,0 +1,120 @@
+
+.. code-block:: json
+		
+   {
+       "ngf": {"_type":"choice","_value":[16, 32, 64, 128, 256]},
+       "ndf": {"_type":"choice","_value":[16, 32, 64, 128, 256]},
+       "netG": {"_type":"choice","_value":["resnet_9blocks", "unet_256"]},
+       "netD": {"_type":"choice","_value":["basic", "pixel", "n_layers"]},
+       "norm": {"_type":"choice","_value":["batch", "instance", "none"]},
+       "init_type": {"_type":"choice","_value":["xavier", "normal", "kaiming", "orthogonal"]},
+       "lr":{"_type":"choice","_value":[0.0001, 0.0005, 0.001, 0.005, 0.01, 0.1]},
+       "beta1":{"_type":"uniform","_value":[0, 1]},
+       "lr_policy": {"_type":"choice","_value":["linear", "step", "plateau", "cosine"]},
+       "gan_mode": {"_type":"choice","_value":["vanilla", "lsgan", "wgangp"]} ,
+       "lambda_L1": {"_type":"choice","_value":[1, 5, 10, 100, 250, 500]}
+    }
+
+Implemented code directory: :githublink:`search_space.json <examples/trials/pix2pix-pytorch/search_space.json>`
+
+Note: starting from v2.0, the search space is directly included in the config, please find the example here: :githublink:`config_v2.yml <examples/trials/pix2pix-pytorch/config_v2.yml>`
+
+**Trial**
+
+To experiment on this set of hyperparameters using NNI, we have to write a trial code, which receives a set of parameter settings from NNI, trains a generator and discriminator using these parameters, and then reports the final scores back to NNI. Then, in the experiment, NNI can repeatedly call this trial code with different set of parameter settings. It is important that the following three lines are incorporated in the trial code: 
+
+* Use ``nni.get_next_parameter()`` to get next hyperparameter set.
+* (Optional) Use ``nni.report_intermediate_result(score)`` to report the intermediate result after finishing each epoch.
+* Use ``nni.report_final_result(score)`` to report the final result before the trial ends.
+
+Implemented code directory: :githublink:`main.py <examples/trials/pix2pix-pytorch/pix2pix.py>`
+
+Some notes on the implementation:
+
+* The trial code for this example is adapted from the `repository maintained by the authors of Pix2pix and CycleGAN <https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix>`__ .  You can also use your previous code directly, refer to `How to define a trial <Trials.rst>`__ for modifying the code.
+* By default, the code uses the dataset "facades". It also supports the datasets 'night2day', 'edges2handbags', 'edges2shoes', and 'maps'.
+* For 'facades', 200 epochs are enough for the model to converge to a point where the difference between different parameters are salient enough for evaluation. If you are using other datasets, please consider increasing the ``n_epochs`` and ``n_epochs_decay`` parameters by either passing them as arguments when calling ``pix2pix.py`` in the config file (discussed below) or changing the ``pix2pix.py`` directly.
```

Comment:
Do you mean: 200 epochs for search to tell the differences between hyper-parameters. For final training, we need more epochs?

Reply:
Yes, although for the "facades" dataset ~200 epochs is enough for the final training too.